Makefile

```diff
@@ -18,7 +18,7 @@ UNIT_TAGS?=unit
 INTEGRATION_TAGS?=integration
 E2E_TAGS?=e2e
 E2E_BINARY?=../../bin/${BINARY_NAME}
-E2E_TIMEOUT?=20m
+E2E_TIMEOUT?=30m
```

Comment:
This is now matching the timeout that we use for the evergreen tasks

Reply:
you mean evergreen as a whole? consider that you want the test to fail before evergreen shuts it down so you can get test results

Reply:
Yes, [evergreen.yml](https://github.com/mongodb/mongocli/blob/f238209efda0fd98b5bad6535da6746db0173e8a/build/ci/evergreen.yml#L15). I am going to decrease this to 25.